lib/active_merchant/billing/gateways/adyen.rb

```diff
@@ -515,14 +515,18 @@ def add_billing_address(post, options, address)
         post[:billingAddress][:postalCode] = address[:zip] if address[:zip]
         post[:billingAddress][:city] = address[:city] || 'NA'
         post[:billingAddress][:stateOrProvince] = get_state(address)
-        post[:billingAddress][:country] = address[:country] if address[:country]
+        post[:billingAddress][:country] = get_country(address)
         post[:telephoneNumber] = address[:phone_number] || address[:phone] || ''
       end
 
       def get_state(address)
         address[:state] && !address[:state].blank? ? address[:state] : 'NA'
       end
 
+      def get_country(address)
+        address[:country] && !address[:country].blank? ? address[:country] : 'ZZ'
```

Comment:
Nitpick so feel no obligation to change: but could this be simplified to the following? 
```suggestion
        address[:country].present? ? address[:country] : 'ZZ'
```

Reply:
Much simpler, I like it! Updated 🎉